app/Repositories/Transactions.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Models\Transaction;
+use App\Repositories\Contracts\TransactionRepository;
+use Carbon\Carbon;
+use Closure;
+
+class Transactions extends Base implements TransactionRepository
+{
+    /**
+     * Determines the model.
+     *
+     * @return mixed
+     */
+    public function model()
+    {
+        return Transaction::class;
+    }
+
+    /**
+     * Get all entries.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function all(...$args)
```

Comment:
So, I use this operator a lot, because for some services I need flexibility for the service instances to pass various arguments instead of hardcoding the expected arguments. So, it's a useful feature for some cases.